README.md

```diff
@@ -275,4 +316,140 @@ We'll be looking at:
 - 🎨 User experience and interface design (where applicable)
 - 💡 Innovation and creativity (blow our minds!)
 
+## 👍 Good Example
+
+You don't have to do all (or any) of these things, but these are some ways you can satisfy the requirements.
+
+- Create a blog post, web page, or public GitHub gist that showcases the work you've completed, including links to the commits, repositories, and pull requests you've contributed to. This approach is ideal for sharing comprehensive details about your work, making it easy for others to review and understand your contributions.
+
+- If using GitHub, and all of your work is contained within a single pull request, link directly to that PR. Ensure that the pull request description is thorough, highlighting the specific tasks completed during the project.
+
+For a GitHub repository created, consider adding a detailed README.md file. This will provide additional context and clarity for anyone reviewing your work.
+
+## 👎 Bad Example
+
+Heres more examples of things you shouldn't do:
+
+- Don't upload a big zip file with all your project files or source code. It's overwhelming and doesn't help people understand your work.
+
+- Don't just link to the main page of the project's source code repository. For example, linking to the top page of Python's GitHub repository isn't helpful.
+
+- Avoid linking to your personal copy of the project's source code. It mixes your changes with others and makes it hard to see what you've done.
+
+- Link to a huge log of all the project’s commits. This just floods the reader with too much information and doesn't highlight your specific contributions.
+
+- Share a screenshot of your code instead of the actual code. This makes it impossible for others to review or use your work effectively.
+
+- Link to a personal blog that just mentions you worked on the project without any real details. This doesn’t provide any useful information about your contributions.
+
+- Upload your code as a PDF file. This makes it hard to review, edit, or understand, especially for larger projects.
+
+- Share a link to a random page in the project’s documentation. This doesn’t show your specific work and just confuses people.
+
+- Link to a massive text file with no formatting or context. This is overwhelming and makes it difficult to find your relevant contributions. 
+
+## ⚙️ Git Workflow
```

Comment:
I don't understand the point of having the git workflow on the bounty page.

Reply:
i added the git workflow so it helps ensure all participants follow similar best practices and code quality, making it easier to review process when they submit their project repo. i remove it.